### PR TITLE
fix: main help button from edit toolbar was broken

### PR DESCRIPTION
**Pull request by yohanboniface**

### CODE REVIEW

File: umap/static/umap/js/modules/help.js [hunk 1]
```diff
@@ -191,7 +191,7 @@ export default class Help {
     const container = DomUtil.add('div')
     DomUtil.createTitle(container, translate('Help'))
     // Special dynamic case. Do we still think this dialog is usefull ?
-    if (entries === 'edit') {
+    if (entries[0] === 'edit') {
```
Comment (davidbgk) [inline umap/static/umap/js/modules/help.js:194]: That exception looks smelly, can't we have a dedicated `showHelp()`?

**Review by davidbgk — APPROVED** [review]

File: umap/static/umap/js/modules/help.js (hunk 1 above)
Comment (yohanboniface) [reply umap/static/umap/js/modules/help.js]: Done! :)

File: umap/static/umap/js/modules/help.js [hunk 2]
```diff
@@ -190,42 +190,48 @@ export default class Help {
...
+    })
     return button
   }
 
-  link(container, entries) {
-    const button = this.button(container, entries, 'umap-help-link')
+  getStartedLink(container) {
+    const button = DomUtil.createButton('umap-help-link', container, translate('Help'))
     button.textContent = translate('Help')
+    DomEvent.on(button, 'click', DomEvent.stop).on(button, 'click', () => {
```
Comment (davidbgk) [inline umap/static/umap/js/modules/help.js:232]: What are we stopping prior to show the element? Would it be easier to use regular JS event handling?

File: umap/static/umap/js/modules/help.js [hunk 3]
```diff
@@ -190,42 +190,48 @@ export default class Help {
...
-          innerHTML: ENTRIES[name],
-        })
-      }
+        innerHTML: ENTRIES[name],
+      })
     }
     this.map.dialog.open({ content: container, className: 'dark' })
   }
 
+  // Special dynamic case. Do we still think this dialog is usefull ?
```
Comment (davidbgk) [inline umap/static/umap/js/modules/help.js:204]:
```suggestion
  // Special dynamic case. Do we still think this dialog is useful?
```

Let's do that in another PR.
